views/partials/safeAnnouncementInfo.ejs

```diff
@@ -0,0 +1,3 @@
+<div class="rf-callout rf-fi-information-line rf-callout--scheme-soft-blue-soft rf-mb-3w">
```

Comment:
```suggestion
<!-- 
Use this template to display info messages containing HTML.
WARNING : this template does not escape HTML, so be sure that no user-provided data is displayed in it, to avoid injection attacks. (google "server side template injection" to learn about it)
-->
<div class="rf-callout rf-fi-information-line rf-callout--scheme-soft-blue-soft rf-mb-3w">
```